src/PlaneWaveBasis.jl

```diff
@@ -62,6 +62,9 @@ struct PlaneWaveBasis{T <: Real}
     # ksymops[ik] is a list of symmetry operations (S,τ)
     # mapping to points in the reducible BZ
     ksymops::Vector{Vector{SymOp}}
+    # Monkhorst-Pack grid used to generate the k-Points, nothing if unknown
```

Comment:
not really unknown, more "custom", non-MP set of kpoints?

Reply:
yes, that's a good point.